examples/8a_benchmark_write_parallel.cpp

```diff
@@ -339,6 +339,8 @@ class TestInput
   unsigned long  m_XFactor = 0; // if not overwritten, use m_MPISize
   unsigned long  m_YFactor = 8;
   unsigned long  m_ZFactor = 8;
+
+  std::string m_Prefix = "../samples";
```

Comment:
Let's add a doxygen string here:
```suggestion
  //! prefix for the output directory
  std::string m_Prefix = "../samples";
```